api/Controllers/AddressesController.cs

```diff
@@ -0,0 +1,41 @@
+using Microsoft.AspNetCore.Mvc;
+using DTO.Address;
+
+
+
+[Route("api/[controller]")]
+[ApiController]
+public class AddressesController : ControllerBase
+{
+    private readonly AddressProvider _addressProvider;
+
+    public AddressesController(AddressProvider addressProvider)
+    {
+        _addressProvider = addressProvider;
+    }
+
+    [HttpPost]
+    public IActionResult Create([FromBody] AddressRequest req)
+    {
+        Address? newAdress = _addressProvider.Create(req);
+
+        if (newAdress == null) throw new ApiFlowException("Saving new Address failed.");
+
+        return Ok(new 
+        {
+            message = "Address created",
+            new_address = new AddressResponse
+            {
+                Id = newAdress.Id,
```

Comment:
zorg ervoor dat de lijnen op de zelfde lijn zitten, dus niet dat de ene voor is en de andere achter